data_validation/config_manager.py

```diff
@@ -748,11 +748,32 @@ def build_and_append_pre_agg_calc_config(
         return calculated_config
 
     def append_pre_agg_calc_field(
-        self, source_column, target_column, agg_type, column_type, column_position
+        self,
+        source_column: str,
+        target_column: str,
+        agg_type: str,
+        column_type: str,
+        target_column_type: str,
+        column_position: int,
     ) -> dict:
         """Append calculated field for length(string | binary) or epoch_seconds(timestamp) for preprocessing before column validation aggregation."""
```

Comment:
```suggestion
        """Append calculated field for length() or epoch_seconds(timestamp) for preprocessing before column validation aggregation."""
```
Removing this data type list otherwise we would need to add the others supported by the method